rtengine/PF_correct_RT.cc

```diff
@@ -1341,7 +1341,7 @@ SSEFUNCTION void ImProcFunctions::BadpixelsLab(LabImage * src, LabImage * dst, d
 #ifdef __SSE2__
         int j;
 //  vfloat2 sincosvalv;
-//  __m128 piidv = _mm_set1_ps(piid);
```

Comment:
It was touced by replacing all _mm_set1_ps( with F2V( in that file. I'll treat this function in a separate issue.